app/views/questionnaire.py

```diff
@@ -40,21 +41,26 @@
                                     import_name=__name__,
                                     url_prefix='/questionnaire/<eq_id>/<form_type>/<collection_id>/')
 
+thank_you_blueprint = Blueprint(name='thank_you',
+                                import_name=__name__,
+                                url_prefix='/thank-you')
+
 
 @questionnaire_blueprint.before_request
 def before_request():
     metadata = get_metadata(current_user)
-    if metadata:
-        logger.bind(tx_id=metadata['tx_id'])
+    if not bool(metadata):
```

Comment:
Why do we need to cast this to a bool? can't we check not None?

Reply:
Or even, why would it ever be None?

Reply:
This is after submission when trying to get back into the questionnaire. get_metadata() returns an empty dict as the user is now still logged in but the metadata has been cleared. I'll look at the logic around returning none instead of an empty dict.